deployment/deployment_test.go

```diff
@@ -1059,3 +1059,42 @@ func TestGetDeploymentNamespaceNameError(t *testing.T) {
 	assert.Equal(t, "", name)
 	assert.EqualError(t, err, "no kubernetes namespaces specified")
 }
+
+func TestAddDagDeploymentArgs(t *testing.T) {
+	tests := []struct {
+		dagDeploymentType string
+		nfsLocation       string
+		sshKey            string
+		knownHosts        string
+		gitRepoURL        string
+		gitRevision       string
+		gitBranchName     string
+		gitDAGDir         string
+		gitSyncInterval   int
+		expectedError     string
+		expectedOutput    map[string]interface{}
+	}{
+		{dagDeploymentType: imageDeploymentType, expectedError: "", expectedOutput: map[string]interface{}{"dagDeployment": map[string]interface{}{"type": imageDeploymentType}}},
+		{dagDeploymentType: volumeDeploymentType, nfsLocation: "test", expectedError: "", expectedOutput: map[string]interface{}{"dagDeployment": map[string]interface{}{"type": volumeDeploymentType, "nfsLocation": "test"}}},
+		{
+			dagDeploymentType: gitSyncDeploymentType,
+			sshKey:            "../cmd/testfiles/ssh_key",
+			knownHosts:        "../cmd/testfiles/known_hosts",
+			gitRepoURL:        "https://github.com/neel-astro/private-airflow-dags-test",
+			gitRevision:       "test-revision", gitBranchName: "test-branch", gitDAGDir: "test-dags",
```

Comment:
might want to do each property on a separate lines for readability purposes

Reply:
updated